jack.py

```diff
@@ -2027,6 +2035,15 @@ def register(self, shortname, is_terminal=False, is_physical=False):
         self._portlist.append(port)
         return port
 
+    def unregister_range(self, start=None, end=None, step=1):
```

Comment:
i wasnt sure if jack.py that was part of the lib or a wrapper you had written, I'll take out the changes for now :+1: